docs/workflow/testing/libraries/testing.md

```diff
@@ -22,7 +22,7 @@ libraries -test /p:WithoutCategories=IgnoreForCI
 
 ## Running tests on the command line
 
-To build tests you need to pass the `-buildtests` flag to build.cmd/sh or if you want to build both src and tests you pass `-buildtests` flag (`libraries -restore -build -buildtests`). Note that you need to specify -restore and -build additionally as those are only implicit if no action is passed in.
+To build tests you need to pass the `-buildtests` flag to `build.cmd/sh` or if you want to build both src and tests you pass `-buildtests` flag (`libraries -restore -build -buildtests`). Note that you need to specify `-restore` and `-build` additionally as those are only implicit if no action is passed in.
```

Comment:
The first part of the sentence says to pass the -buildtests flag... the second part of the sentence says if you want to build both src and tests to pass the -buildtests flag.  Those are the same.  What am I missing?

Reply:
Good point, I have updated the text in my latest commits.